optimum/bettertransformer/transformation.py

```diff
@@ -98,10 +100,13 @@ def set_last_layer(model: torch.nn.Module):
             isinstance(dict_named_module[key], torch.nn.ModuleList)
             and "encoder" in key
             and (
-                model.config.model_type not in EXCLUDE_FROM_TRANSFORM
+                model.config.model_type not in BetterTransformerManager.EXCLUDE_FROM_TRANSFORM
                 or (
-                    model.config.model_type in EXCLUDE_FROM_TRANSFORM
-                    and all(name not in key for name in EXCLUDE_FROM_TRANSFORM[model.config.model_type])
+                    model.config.model_type in BetterTransformerManager.EXCLUDE_FROM_TRANSFORM
+                    and all(
+                        name not in key
+                        for name in BetterTransformerManager.EXCLUDE_FROM_TRANSFORM[model.config.model_type]
+                    )
                 )
             )
```

Comment:
I do not think that it is easy to read, but feel free to keep it like that!

Reply:
I'll leave as is but I agree